configs/projects/puppet-agent.rb

```diff
@@ -89,6 +89,7 @@
   end
 
   # First our stuff
+  proj.component "leatherman"
```

Comment:
Yup. I moved it up because I forgot to save my changes to the facter file and was very confused about why my new build dependency wasn't making leatherman build first -_-